### DISCUSSION TITLE
How can a user select and delete a link when Segmentable == true?

### DISCUSSION THREAD

**Discussion by JohnMidity** [Blazor-Diagrams/Blazor.Diagrams#404] [Q&A]
Thank you for this amazing library. 
I use StraightPathGenerator and Segmentable in my diagram. When clicking on a line, a  new vertex is added. Because the link cannot be selected, it can also not be removed. Any way around this? Is there a hidden keycombination or some other solution?

**Comment by zHaytam** [comment]
Hello, can you create an isssue from this discussion please?

**Comment by JohnMidity** [comment]
Created issue #419
